src/components/layout/GradientBottomActions.tsx

```diff
@@ -99,7 +101,12 @@ export const GradientBottomActions = ({
       ]}
     />
     <View style={styles.buttonContainer} pointerEvents="box-none">
-      {primaryAction}
+      {primaryAction &&
+        (primaryAction.type === "ButtonLink" ? (
```

Comment:
We don't need this condition because the primary action is always a `ButtonSolid` and the secondary action is always a `ButtonLink`.

Reply:
Ah ok. I believe that both the primary and the secondary action could be a ButtonSolid or a ButtonLInk. I'll fix it. Thanks